apps/server/src/modules/authentication/strategy/x-api-key.strategy.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable, UnauthorizedException } from '@nestjs/common';
+import { PassportStrategy } from '@nestjs/passport';
+import { ConfigService } from '@nestjs/config';
+import Strategy from 'passport-headerapikey';
+import { IXApiKeyConfig } from '../config/x-api-key.config';
+
+@Injectable()
+export class XApiKeyStrategy extends PassportStrategy(Strategy, 'api-key') {
+	private readonly allowedApiKeys: string[];
+
+	constructor(private readonly configService: ConfigService<IXApiKeyConfig, true>) {
+		super({ header: 'X-API-KEY', prefix: '' }, true);
```

Comment:
Just curious, is `prefix` needed to be specified here if it's empty?

Reply:
probably not